### ISSUE TITLE
Simplifying Specification Metadata - don't hide on Editor's drafts

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/tr-design#140]
I'm wondering if hiding the metadata should only happened on published specs? I'm super reliant on those links in my day-to-day work (with Editor's drafts) and having to open the `<details>` is going to be a hindrance.

**Comment by silviapfeiffer** [comment]
I agree. I don't like hiding those important links to the repo, older versions and the editor draft. If I hadn't known they were there, I certainly wouldn't have looked for them under the "Metadata" label.

**Comment by plehegar** [comment]
The proposed change is only applicable to published specs. We don't run pubrules on Editor's drafts. In addition, kind of hidden in the proposal is the use of Web Storage to see the details by default instead of having them hidden. Ie advanced users could set up their preferences to always see details.
https://rawgit.com/w3c/tr-design/versioning/spec-preferences.html (and reload the spec sample after that selecting visible)
